src/System.Private.Uri/src/System/UriBuilder.cs

```diff
@@ -114,19 +81,12 @@ string extraValue
             {
                 Extra = extraValue;
             }
-            catch (Exception exception)
+            catch (ArgumentException)
```

Comment:
Sure, I'll revert this. Should I revert the other changes to UriBuilder Init as well?